code/testapp/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.adobe.mediaanalyticstestapp">
+    package="com.adobe.edge.media.testapp">
 
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
 
     <application
         android:networkSecurityConfig="@xml/network_security_config"
-        android:name=".MediaAnalyticsTestApp"
+        android:name=".MediaTestApp"
```

Comment:
nit: Should we name it EdgeMediaTestApp

Reply:
This name needs to match the Application class name.